src/components/combo-box/combo-box.hbs

```diff
@@ -129,10 +247,50 @@
     </div>
   </div>
 </div>
+<div style="margin-bottom:10rem"></div>
```

Comment:
Instead of adding a markup for spacer, could we do a devenv-specific style?

Reply:
yes I've made that change now